source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+  font-size: 1rem; // 16px
+
+  hx-def {
+    display: flex;
+
+    hx-dt {
+      #grid.span(2);
+      color: @gray-750;
+      display: block;
+      flex-shrink: 1;
+      font-weight: 400;
+      text-align: right;
+    }
+
+    hx-dd {
+      #grid.span(10);
+      color: @gray-900;
```

Comment:
The property is unnecessary. Body copy is [already at gray 900](https://github.com/rackerlabs/helix-ui/blob/master/source/components/typography/typography.less#L2).